src/core/operations/CipherSaber2.mjs

```diff
@@ -0,0 +1,98 @@
+/**
+ * @author n1073645 [n1073645@gmail.com]
+ * @copyright Crown Copyright 2020
+ * @license Apache-2.0
+ */
+
+import Operation from "../Operation.mjs";
+import crypto from "crypto";
+import Utils from "../Utils.mjs";
+
+/**
+ * CipherSaber2 operation
+ */
+class CipherSaber2 extends Operation {
+
+    /**
+     * CipherSaber2 constructor
+     */
+    constructor() {
+        super();
+
+        this.name = "CipherSaber2";
+        this.module = "Crypto";
+        this.description = "";
+        this.infoURL = "";
+        this.inputType = "ArrayBuffer";
+        this.outputType = "ArrayBuffer";
+        this.args = [
+            {
+                name: "Key",
+                type: "string",
+                value: ""
+            },
+            {
+                name: "Rounds",
+                type: "number",
+                value: 20
+            },
+            {
+                name: "Mode",
+                type: "option",
+                value: ["Encrypt", "Decrypt"]
```

Comment:
I'd prefer separate operations for encrypt and decrypt, in line with other cipher ops.

Reply:
It probably makes sense to pull the code out into a library so you don't duplicate it in each op.